packages/fxa-content-server/app/scripts/views/subscriptions_product_redirect.js

```diff
@@ -25,13 +25,18 @@ class SubscriptionsProductRedirectView extends FormView {
   }
 
   afterRender() {
-    const searchQuery = this.window.location.search;
+    const locationUrl = new URL(this.window.location.href);
+    const queryParams = {};
+    for (const [key, value] of locationUrl.searchParams.entries()) {
```

Comment:
We probably shouldn't use `for of` and within the content server, we don't have all the goodness set up in babel. To get the query params, you should be able to call `Url.searchParams(this.window.location.href)` w/o needing to use homegrown logic here, see https://github.com/mozilla/fxa/blob/ed501fa1c3485ea33d47ea75ac3b047782fb7bc5/packages/fxa-content-server/app/scripts/lib/url.js#L19